src/dmd/backend/dvarstats.d

```diff
@@ -245,7 +240,7 @@ private symtab_t* calcLexicalScope(symtab_t* symtab) return
         lifeTimes[si].offDestroy = cast(int)getLineOffset(lifeTimes[si].sym.lnoscopeend);
     }
     cntUsedLifeTimes = dupcnt;
-    qsort(lifeTimes, dupcnt, (LifeTime).sizeof, &cmpLifeTime);
+    qsort(lifeTimes.ptr, dupcnt, (lifeTimes[0]).sizeof, &cmpLifeTime);
```

Comment:
Why `lifeTimes[0]` instead of `lifeTime`?

Reply:
Consistency with the other line where the type of the element is used instead of the type.

Reply:
Although I could go with either, as long as we stay consistent.

Reply:
Fair enough